src/gitlab.coffee

```diff
@@ -60,6 +60,16 @@ module.exports = (robot) ->
   trim_commit_url = (url) ->
     url.replace(/(\/[0-9a-f]{9})[0-9a-f]+$/, '$1')
 
+  strip_content = (str) ->
+    str.split(/\n/).filter (line, index) ->
+      # Remove the \n first so we have content only
+      if line.length != 0
+          line
+    .filter (_line, _index) ->
+      if _index <= 4
+        _line
+    .join("\n")
```

Comment:
@spruce the idea behind this, if a comment starts like this

```
Hello
\n
\n
\n

I'm here
```

The bot will just print the `Hello` here, what this `if` make it skip the new line and print on the content. so in this case it will print `Hello\nI'm here`